cmd/monaco/v2/legacy/integration_test_utils.go

```diff
@@ -168,24 +168,39 @@ func cleanupIntegrationTest(t *testing.T, fs afero.Fs, envFile, suffix string) {
 // e.g. my-config_1605258980000_Suffix
 
 func RunLegacyIntegrationWithCleanup(t *testing.T, configFolder, envFile, suffixTest string, testFunc func(fs afero.Fs)) {
-	envvars.InstallFakeEnvironment(map[string]string{
+	configFolder, _ = filepath.Abs(configFolder)
+	envFile, _ = filepath.Abs(envFile)
+
+	envvars.InstallFakeOsOverrideEnvironment(map[string]string{
 		"CONFIG_V1": "1",
 	})
 
 	defer func() {
 		envvars.InstallOsBased()
 	}()
 
-	suffix := getTimestamp() + suffixTest
+	randomNumber := rand.Intn(100)
+
+	suffix := fmt.Sprintf("%s_%d_%s", getTimestamp(), randomNumber, suffixTest)
```

Comment:
The reason for this is to prevent accidental collisions with other
parallel runs of the integration tests. By appending a random number
between 0 and 100 this will become even more unlikely (this was the
reason btw, why we disabled multi os integration tests)